OsiguSDK.Providers/Models/V1/Diagnosis.cs

```diff
@@ -9,13 +9,13 @@ public class Diagnosis
         /// Dianosis ID
         /// </summary>
         [JsonProperty(PropertyName = "id")]
-        public string ProductId { get; set; }
+        public decimal Id { get; set; }
 
         /// <summary>
         /// Diagnosis
         /// </summary>
         [JsonProperty(PropertyName = "name")]
-        public string Name { get; set; }
+        public string Description { get; set; }
```

Comment:
this seems like another breaking change...

Reply:
correct you should stick what is says in the documentation

 {
        "content": [
            { 
                "id": 1,
                "*name*":"Acid Reflux (Gastroesophageal Reflux Disease (GERD))",
            },
            { 
                "id": 1
                "name":"Reglux",
            },
            { 
                "id": 1
                "name":"Reglux",
            }
        ],
        "total_pages":4,
        "number_of_elements":2,
        "total_elements":8,
        "first_page":true,
        "last_page":false,
        "_links":
            {
                "next": {
                    "href": "/v1/diagnoses/search?page=1&size=2"
                },
                "first": {
                    "href": "/v1/diagnoses/search?page=0&size=2"
                },
                "last": {
                    "href": "/v1/diagnoses/search?page=2&size=2"
                },
                "current": {
                    "href": "/v1/diagnoses/search?page=0&size=2"
                }
            }
    }